src/mono/wasm/debugger/BrowserDebugProxy/DebugStore.cs

```diff
@@ -530,6 +562,84 @@ public bool IsLexicallyContainedInMethod(MethodInfo containerMethod)
                     (EndLocation.Line == containerMethod.EndLocation.Line && EndLocation.Column < containerMethod.EndLocation.Column));
     }
 
+    internal class ParameterInfo
+    {
+        public string Name { get; private set; }
+
+        public bool HasDefaultValue { get; private set; }
+
+        public ElementType? TypeCode { get; private set; }
+
+        public object Value { get; set; }
```

Comment:
You can use `init` instead of all `private set` for the properties. It will allow setting the values only in the constructor.